data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,810 @@
+	description `	Although each shot does not hit very hard, consistent fire adds up - but so does energy usage.`
+
+effect "astuit hit"
+	sprite "effect/ast hit"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 6
+	"velocity scale" .001
+
+
+outfit "Acuit Artillery"
+	category "Turrets"
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+	cost 1406000
+	thumbnail "outfit/acuit"
+	"mass" 81
+	"outfit space" -81
+	"weapon capacity" -81
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/acuit"
+			"random start frame"
+		"hardpoint sprite" "hardpoint/acuit"
+		"hardpoint offset" 14
+		"hit effect" "acuit hit"
+		sound "acuit"
+		"inaccuracy" 3
+		"turret turn" 1.3
+		"velocity" 18
+		"lifetime" 60
+		"reload" 56
+		"burst reload" 10
+		"burst count" 1
+		"firing energy" 130
+		"firing heat" 204
+		"shield damage" 170
+		"hull damage" 290
+		"hit force" 40
+	description `With a better grasp on plasma and energy weaponry, the Gegno Scin developed a larger blaster weapon to mount on the Feldspar Carrier as a counter to the Gegno Vi's huge cannon weaponry. The Acuit Artillery Cannon outranges most weapons in Gegno space, and has a rate of fire higher than most other weapons, making it a very efficient bombardment weapon from afar.`
+
+effect "acuit hit"
+	sprite "effect/acuit hit"
+		"frame rate" 18
+		"no repeat"
+	"lifetime" 18
+	"velocity scale" -0.004
+
+
+outfit "Guile Pulse Laser"
+	category "Guns"
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+	cost 109000
+	"thumbnail" "outfit/guile pulse laser"
+	"mass" 15
+	"outfit space" -15
+	"weapon capacity" -15
+	"gun ports" -1
+	weapon
+		"sprite" "projectile/guile pulse laser"
+			"frame rate" 6
+		sound "guile pulse"
+		"hit effect" "heavy laser impact"
+		"inaccuracy" 0.2
+		"velocity" 150
+		"lifetime" 1
+		"reload" 6
+		"burst count" 3
+		"burst reload" 1
+		"firing energy" 3.7
+		"firing heat" 10
+		"shield damage" 12.4
+		"hull damage" 20.2
+		"cluster"
+	description `The Guile Pulse Laser is a recent experimental advancement in Scin plasma weapon technology. Rather than rapid firing plasma slugs, this weapon fires concentrated pulses of energy that only last for a brief moment. Each blast impacts the target ship almost instantaneously with a powerful hit, and a consistent amount of fire from Guile Lasers is extremely effective against slow moving targets.` 
+	description `	Guile Pulse Lasers have noticeable drawbacks as they quickly consume energy and generate notable heat per shot. Unlike a beam-type weapon, they are incapable of consistent output, and have a fairly short range.`
+
+
+outfit "Excavator"
+	category "Guns"
+	cost 0
+	"thumbnail" "outfit/mouthparts"
+	"mass" 42
+	"outfit space" -42
+	"weapon capacity" -42
+	"gun ports" -1
```

Comment:
Would it be best to give the Gypsum some type of attribute that would be negative and the positive part on the Excavator? Maybe even use "Excavator" as the attribute? 

I'm assuming this would still show up in the outfitter if you had one though, just not sellable? (Versus showing up as purchasable.) (Is there perhaps a way to hide an outfit from the outfitter completely? Or if you have one will it always show?)

Reply:
```
ship:
  "excavator port" 1
  "excavator lock" -1
weapon:
  "excavator port" -1
  "excavator lock" 1
```